### PR TITLE
update installer win

### PR DESCRIPTION
**Pull request by cbielow**
## Describe the change

Make user aware of Windows Defender when using our installer.

(this PR is against develop -- should it be staging? (seems outdated, but the PR pretext says so...)

## PR checklist

- [ ] I have added description of the change I'm proposing in the OpenMS Documentation.
- [ ] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

**Comment by jpfeuffer** [comment]
I think we ditched staging. can you update the text?

File: docs/openms-applications-and-tools/installation/installation-on-windows.md [hunk 1]
```diff
@@ -5,13 +5,19 @@ Windows
 
 To Install the binary package of OpenMS & {term}`TOPP`:
 
-1. Download and execute the installer `OpenMS-<version>-Win64.exe` from the [archive](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/) and follow its instructions.
-2. Run the installer under the user account that later runs OpenMS. It is not advised to use admin account for
-   installation. When asked for an admin authentication, please enter the credentials.
+1. Download the installer `OpenMS-<version>-Win64.exe` from the [archive](https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/latest/) 
+2. Execte the installer under the user account that later runs OpenMS and follow its instructions.
```
Comment (jpfeuffer) [inline docs/openms-applications-and-tools/installation/installation-on-windows.md]:
```suggestion
2. Execute the installer under the user account that later runs OpenMS and follow its instructions.
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Yes staging is obsolete. Develop is the main branch I believe.

**Review by ghost — APPROVED** [review]
No issues from my end.

**Pull request merged**